examples/connectivity/plot_atlas_comparison.py

```diff
@@ -0,0 +1,51 @@
+"""
+Extracting signals from different atlases and compare
+======================================================
+
+In this example :class:`nilearn.input_data.NiftiLabelsMasker` is used to
+extract time series from nifti objects using different parcellation atlases.
+
+The time series of all subjects of the ADHD Dataset are concatenated to create
+parcel-wise correlation matrices for each atlas.
+
+"""
+
+from nilearn import datasets
+from nilearn.input_data import NiftiLabelsMasker
+import numpy as np
+from matplotlib import pyplot as plt
+
+# load atlases
+destrieux = datasets.fetch_atlas_destrieux_2009()
+yeo = datasets.fetch_atlas_yeo_2011()
+harvard_oxford = datasets.fetch_atlas_harvard_oxford('cort-maxprob-thr25-2mm')
+
+atlases = {'Destrieux Atlas': destrieux['maps'],
+           'Yeo Atlas 17 thick': yeo['thick_17'],
+           'Harvard Oxford > 25%': harvard_oxford['maps']}
```

Comment:
From a neuroscientific perspective it might be intersting to perhaps add in brackets in the dictionary keys/titles whether each is a structural atlas (e.g. Harvard Oxford) or a functional atlas (eg. Yeo?)